microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -128,29 +162,192 @@ Version 1.1.8 of the [Microsoft Edge Developer Tools for Visual Studio Code][Vis
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1192084][CR1192084].
+
+
+### Enhanced CORS debugging support
+
+Cross-origin resource sharing (CORS) errors are now surfaced in the **Issues** tab.  There are various potential causes of CORS errors.  Click to expand each issue to understand the potential causes and solutions.
+
+:::image type="complex" source="../../media/2021/05/cors-debugging-support.msft.png" alt-text="CORS issues in the Issues tab" lightbox="../../media/2021/05/cors-debugging-support.msft.png":::
+   CORS issues in the Issues tab
+:::image-end:::
+
+<!-- screenshot uses http://cors-errors.glitch.me -->
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1141824][CR1141824].
+
+
+### Renamed XHR label to Fetch\/XHR
+
+In the **Network** tool, the **XHR** label is now renamed to **Fetch/XHR**. This change makes it clearer that this filter includes both `XMLHttpRequest` and `Fetch` API network requests.
+
+:::image type="complex" source="../../media/2021/05/fetch-xhr.msft.png" alt-text="The Network tool now shows Fetch/XHR instead of XHR" lightbox="../../media/2021/05/fetch-xhr.msft.png":::
+   The **Network** tool now shows **Fetch/XHR** instead of **XHR**
+:::image-end:::
+
+For more information, navigate to:
+*  [XMLHttpRequest spec][XhrSpecWhatwgOrg]
+*  [Fetch spec][FetchSpecWhatwgOrg]
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1201398][CR1201398].
+
+
+### Filter Wasm resource type in the Network tool
+
+In the **Network** tool, you can now select the new **Wasm** button to filter the WebAssembly network requests.
+
+:::image type="complex" source="../../media/2021/05/wasm-network-requests.msft.png" alt-text="Filter by Wasm" lightbox="../../media/2021/05/wasm-network-requests.msft.png":::
+   Filter by Wasm
+:::image-end:::
+
+<!-- screenshot uses http://memory-inspector.glitch.me/demo-wasm.html -->
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1103638][CR1103638].
+
+
+### Compute Intersections are now included in the Performance tool
+
+In the **Performance** tool, DevTools now displays **Compute Intersections** in the flame chart. These changes help you identify intersection observers events and debug the potential performance overhead of intersection observers.
+
+:::image type="complex" source="../../media/2021/05/compute-intersections-in-perf-tool.msft.png" alt-text="Compute Intersections in the Performance tool" lightbox="../../media/2021/05/compute-intersections-in-perf-tool.msft.png":::
+   Compute Intersections in the **Performance** tool
+:::image-end:::
+
+<!-- screenshot uses https://googlechrome.github.io/samples/intersectionobserver -->
+
+For more about intersection observers, navigate to [Trust is good, observation is better: Intersection Observer v2][WebDevIntersectionObserverV2].
```

Comment:
Instead of having these as 3 separate lines, can we just merge them all into one paragraph?